docs/buildah-bud.md

```diff
@@ -30,6 +37,73 @@ resulting image's configuration.
 Use certificates at *path* (*.crt, *.cert, *.key) to connect to the registry.
 Default certificates directory is _/etc/containers/certs.d_.
 
+**--cgroup-parent**=""
```

Comment:
An example for this one would be good too.

Reply:
fixed